src/stylesheets/theme/_uswds-theme-components.scss

```diff
@@ -100,6 +100,7 @@ $theme-identifier-font-family: "ui";
 $theme-identifier-background-color: "base-darkest";
 $theme-identifier-identity-domain-color: "base-light";
 $theme-identifier-max-width: "desktop";
+$theme-identifier-primary-link-color: "base-lighter";
```

Comment:
Created this new setting for the parent agency link so users can more easily override.